use_cases/asynchronous_mail_send.md

```diff
@@ -1,15 +1,35 @@
 # Asynchronous Mail Send
 
-## Using `asyncio` (3.5+)
+## Using `asyncio` (3.6+)
 
 The built-in `asyncio` library can be used to send email in a non-blocking manner. `asyncio` helps us execute mail sending in a separate context, allowing us to continue the execution of business logic without waiting for all our emails to send first.
 
 ```python
 from sendgrid import SendGridAPIClient
 from sendgrid.helpers.mail import Content, Mail, From, To, Mail
+from functools import partial
 import os
 import asyncio
 
+try:
+    # Python 3.7+ only.  This is more efficient than get_event_loop where
+    # available.
+    from asyncio import get_running_loop
+    from asyncio import run as async_run
+except ImportError:
+    # Python 3.6+ compatibility
+    from asyncio import get_event_loop as get_running_loop
+
+    def async_run(future):
```

Comment:
Should this be within the except? Or top level?

Reply:
Within the except.  `asyncio.run` is imported as `async_run`, but `asyncio.run` only exists for Python 3.7.  The version within the except is a rough implementation of `asyncio.run` for when it's absent (which would be the case for Python 3.6).